meerkat/columns/text_column.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import annotations
+
+from typing import Sequence
+
+from meerkat.columns.list_column import ListColumn
+
+
+class TextOutputColumn(ListColumn):
```

Comment:
Is there any functionality that is different from a `ListColumn`? Maybe we can punt this for now?

Reply:
Yes, for now this is redundant. I'll remove this